diff --git a/home/naftuli/.local/share/nvim/plugged/vim-go/autoload/go/complete.vim b/complete.vim
index 127d57f..279b10c 100644
--- a/home/naftuli/.local/share/nvim/plugged/vim-go/autoload/go/complete.vim
+++ b/complete.vim
@@ -9,6 +9,7 @@ function! s:gocodeCommand(cmd, args) abort
   let cmd = [bin_path]
   let cmd = extend(cmd, ['-sock', socket_type])
   let cmd = extend(cmd, ['-f', 'vim'])
+  let cmd = extend(cmd, ['-source'])
   let cmd = extend(cmd, [a:cmd])
   let cmd = extend(cmd, a:args)
 
